esdt-safe/tests/blackbox_setup.rs

```diff
@@ -0,0 +1,336 @@
+use bls_signature::BlsSignature;
+use esdt_safe::esdt_safe_proxy::{self};
+use fee_market::fee_market_proxy::{self, FeeStruct};
+use header_verifier::header_verifier_proxy;
+use multiversx_sc::codec::TopEncode;
+use multiversx_sc::types::{
+    Address, EsdtTokenData, ManagedByteArray, MultiValueEncoded, TestTokenIdentifier,
+};
+use multiversx_sc::{
+    imports::{MultiValue3, OptionalValue},
+    types::{BigUint, EsdtTokenPayment, ManagedBuffer, ManagedVec, TestAddress, TestSCAddress},
+};
+use multiversx_sc_scenario::multiversx_chain_vm::crypto_functions::sha256;
+use multiversx_sc_scenario::{api::StaticApi, imports::MxscPath, ScenarioTxRun, ScenarioWorld};
+use multiversx_sc_scenario::{managed_address, ReturnsHandledOrError};
+use transaction::{Operation, OperationData, OperationEsdtPayment};
+
+const BRIDGE_ADDRESS: TestSCAddress = TestSCAddress::new("bridge");
+const BRIDGE_CODE_PATH: MxscPath = MxscPath::new("output/esdt-safe.mxsc.json");
+const BRIDGE_OWNER_ADDRESS: TestAddress = TestAddress::new("bridge_owner");
```

Comment:
I think @andreiblt1304 was naming this `OWNER_ADDRESS`, we should use the same naming conventions across all contracts